### PR TITLE
fix: schema def

### PR DESCRIPTION
**Pull request by jimezesinachi**
# SynthQL pull request template

## Why

Motivation an context for the change.

## What changed

Brief summary of the changes.

## Important

Mention important points that require special attention or that might surprise
reviewers.

### CODE REVIEW

File: packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts [hunk 1]
```diff
@@ -32,6 +32,8 @@ interface PgExecutorProps {
     logging?: boolean;
     /**
      * An SQL statement that will be sent before every SynthQL query.
+     *
+     * e.g `SET search_path TO "public";`
```
Comment (fhur) [inline packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts:36]: Is this a good example? I think messing with the search path can results in bugs. We make some assumptions about your search path being at public.

File: packages/cli/src/cli.ts [hunk 2]
```diff
@@ -85,7 +89,11 @@ export function cli({
                     throw new Error(
                         [
                             `Schema definition overrides does not match the validation schema:`,
-                            ...(validateSchemaDefOverrides.errors ?? []),
+                            ...(JSON.stringify(
```
Comment (fhur) [inline packages/cli/src/cli.ts:92]: JSON.stringify is just a string. Why `?? []` and `...` ?

File: packages/backend/src/tests/propertyBased/tablesToSkip.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const tablesToSkip = [
```
Comment (fhur) [inline packages/backend/src/tests/propertyBased/tablesToSkip.ts:1]: Add a comment explaining why we skip these. We will not remember 1 year from today.

File: packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts (hunk 1 above)
Comment (jimezesinachi) [reply packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts:36]: Changed it to `SELECT version();`

**Review by fhur — APPROVED** [review]

**Pull request merged**